### DISCUSSION TITLE
Back to Top feat

### DISCUSSION THREAD

**Discussion by sudhanya-jaisingh** [imfing/hextra#177] [Q&A] [answered]
I added the i18n folder along with en.yaml file to my project.
Presently my yaml file has the same code as this repos i18n/en.yaml but I'm unable to figure out how to display that feature in my project. I thought just adding the en.yaml was the sol but apparently not. How can I add this feature and the last updated feature in my project?

**Comment by imfing** [comment, marked as answer]
Hi @sudhanya-jaisingh back to top feature is enabled by default. there's no need to configure or add the `en.yaml` which is solely for translation. 

Did you use the following template to bootstrap your site? Note that the scroll to top button shows up when you scroll down a little bit. 
https://github.com/imfing/hextra-starter-template
